ql/instruments/bond.hpp

```diff
@@ -339,6 +351,14 @@ namespace QuantLib {
         return issueDate_;
     }
 
+/*! Returns <tt>true</tt> iff the two bond prices belong to the same
+    derived class.
+    \relates Bond::Price
+*/
+    bool operator==(const Bond::Price&, const Bond::Price&);
```

Comment:
Is this used only to compare with `Null<Price>()`?

Reply:
Yes, only for that. If there's a better way, please show me.

Reply:
We can add something like `price.isValid()`